gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest.py.j2

```diff
@@ -0,0 +1,200 @@
+{% extends '_base.py.j2' %}
+
+{% block content %}
+import warnings
+from typing import Callable, Dict, Optional, Sequence, Tuple
+
+{% if service.has_lro %}
+from google.api_core import operations_v1
+{%- endif %}
+from google.api_core import gapic_v1       # type: ignore
+from google import auth                    # type: ignore
+from google.auth import credentials        # type: ignore
+from google.auth.transport.grpc import SslCredentials  # type: ignore
+
+import grpc  # type: ignore
+
+from google.auth.transport.requests import AuthorizedSession
+
+{# TODO: re-add python_import/ python_modules from removed diff/current grpc template code #}
+{% filter sort_lines -%}
+{% for method in service.methods.values() -%}
+{{ method.input.ident.python_import }}
+{{ method.output.ident.python_import }}
+{% endfor -%}
+{% if opts.add_iam_methods %}
+from google.iam.v1 import iam_policy_pb2 as iam_policy  # type: ignore
+from google.iam.v1 import policy_pb2 as policy  # type: ignore
+{% endif %}
+{% endfilter %}
+
+from .base import {{ service.name }}Transport, DEFAULT_CLIENT_INFO
+
+
+class {{ service.name }}RestTransport({{ service.name }}Transport):
+    """REST backend transport for {{ service.name }}.
+
+    {{ service.meta.doc|rst(width=72, indent=4) }}
+
+    This class defines the same methods as the primary client, so the
+    primary client can load the underlying transport implementation
+    and call it.
+
+    It sends JSON representations of protocol buffers over HTTP/1.1
+    """
+    def __init__(self, *,
+            host: str = 'compute.googleapis.com',
```

Comment:
This should be taken or computed from the service.

Reply:
Oops I accidentally left this in there. Will revert.